### DISCUSSION TITLE
Reactive controller

### DISCUSSION THREAD

**Discussion by andim27** [nylo-core/nylo#51] [Ideas]
Is a way to get reactive controller in Nylo?
It would be nice to have capability like this:
Example:
 - in the page:Text(widget.controller.title)
 - in controller define variable( String title='myTitle)
 - in the page:(Button onPress:widget.controller.changeVariable('title') )
 -  after that widget in the page has changed and we see new title in the Text(Text(widget.controller.title))
The idea is to get access to reactive variables in the controller and immediately update appropriate page.
Such an opportunity allows you to separate the logic for processing variables into pages

**Comment by agordn52** [comment]
Hi @andim27,

Thanks for sharing the idea, I'll see if this is possible.

**Reply by sizoune** [reply]
hi @agordn52 any update about this :D

**Reply by agordn52** [reply]
@sizoune I'll explore this idea a bit more :) I forgot how far I got last time
